core/src/main/java/org/kiwix/kiwixmobile/core/utils/Constants.kt

```diff
@@ -40,6 +40,10 @@ const val TAG_CURRENT_POSITIONS = "currentpositions"
 const val TAG_CURRENT_TAB = "currenttab"
 const val TAG_FROM_TAB_SWITCHER = "fromtabswitcher"
 
+// Http Client
+const val CONNECTION_TIMEOUT: Long = 10
```

Comment:
Don't move these to a constants file. Constants files are generally bad, a constant should be attached to its directly related class.

Consider when you set an android view's visibility, we set it like `myView.setVisibility(View.VISIBLE)`
The constant is stored on the view class because the property is directly related to the view. I shudder to imagine if there was a `Constants` file in the android SDK. They wouldn't be able to reuse the name `VISIBLE` again so everything would probably get prefixed annoyingly.

Reply:
Nice insight!

Reply:
Also, could you please check why is instrumentation coverage failing.
`
> Task :app:compileDebugAndroidTestKotlin FAILED
e: /Users/runner/runners/2.165.2/work/kiwix-android/kiwix-android/app/src/androidTest/java/org/kiwix/kiwixmobile/core/di/modules/TestNetworkModule.kt: (32, 12): 'provideKiwixService' overrides nothing
`

Reply:
`ctrl+shift+n` to TestNetworkModule, it sounds like it isn't overriding anything, you will probably have to declare `provideKiwixService` as `open` in `NetworkModule` because in Kotlin everything is `final` by default, as it should be.

Reply:
Yes, I am aware of that. NetworkModule already has the keyword "open". Trying to figure it out.

Reply:
The function `provideKiwixService` doesn't have the keyword `open`